readyapi4j-core/src/main/java/com/smartbear/readyapi4j/attachments/Attachments.java

```diff
@@ -12,6 +10,10 @@
 import java.net.URLConnection;
 import java.nio.file.Files;
 
+/**
+ * Utility class for creating a RequestAttachmentBuilder for various types of attachments
```

Comment:
A bit too technical perhaps. I suggest:

"Utility class for adding attachments to a request"